src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -24,36 +24,31 @@
 
 package net.fabricmc.loom;
 
-import java.io.File;
-import java.util.List;
-import java.util.Set;
-import java.util.function.Supplier;
-
-import org.cadixdev.lorenz.MappingSet;
-import org.cadixdev.mercury.Mercury;
-import org.gradle.api.NamedDomainObjectProvider;
-import org.gradle.api.Project;
-import org.gradle.api.artifacts.Configuration;
-import org.gradle.api.file.ConfigurableFileCollection;
-
 import net.fabricmc.loom.api.LoomGradleExtensionAPI;
 import net.fabricmc.loom.configuration.InstallerData;
 import net.fabricmc.loom.configuration.LoomDependencyManager;
 import net.fabricmc.loom.configuration.accesswidener.AccessWidenerFile;
 import net.fabricmc.loom.configuration.processors.JarProcessorManager;
 import net.fabricmc.loom.configuration.providers.MinecraftProviderImpl;
-import net.fabricmc.loom.configuration.providers.forge.FieldMigratedMappingsProvider;
-import net.fabricmc.loom.configuration.providers.forge.ForgeProvider;
-import net.fabricmc.loom.configuration.providers.forge.ForgeUniversalProvider;
-import net.fabricmc.loom.configuration.providers.forge.ForgeUserdevProvider;
-import net.fabricmc.loom.configuration.providers.forge.McpConfigProvider;
-import net.fabricmc.loom.configuration.providers.forge.PatchProvider;
-import net.fabricmc.loom.configuration.providers.forge.SrgProvider;
+import net.fabricmc.loom.configuration.providers.forge.*;
```

Comment:
Disable wildcards in your editor to avoid un-necessary changes like seen here.

Reply:
I personally prefer flat imports, since wildcard imports can lead to unwanted sideeffects and name clashes down the line from otherwise non breaking changes. Nevertheless i respect your style choices for this project. I do believe my editor follows your .editorconfig, which also suggests my behaviour (`ij_java_class_count_to_use_import_on_demand=999` - minimum 999 imports to replace with a wildcard import).

Reply:
AH nevermind, my editor did not respect that rule for some reason. Yeah that needs to be changed.